guides/common/modules/proc_uploading-additional-scap-content-using-web-ui.adoc

```diff
@@ -1,7 +1,7 @@
 :_mod-docs-content-type: PROCEDURE
 
-[id="Uploading_Additional_SCAP_Content_{context}"]
-= Uploading additional SCAP content
+[id="uploading-additional-scap-content-using-web-ui"]
+= Uploading additional SCAP content using {ProjectWebUI}
```

Comment:
This seems to be the only instance of "SCAP content" opposed to "SCAP contents".

Reply:
https://github.com/theforeman/foreman-documentation/issues/4214